src/createElement.js

```diff
@@ -0,0 +1,9 @@
+
+export default function createElement(element, attributes, children = '') {
+  const attributesString = attributes ? ` ${Object.entries(attributes).map(([attr, val]) => {
+    if (val == null) return '';
+    if (val === true) return attr;
+    return `${attr}="${val}"`;
```

Comment:
i feel like `sanitizeAttrVal` should go here, and not be the consumer's responsibility